package.json

```diff
@@ -2,6 +2,7 @@
   "name": "react-formal",
   "version": "0.17.9",
   "description": "Classy HTML form management for React",
+  "jsnext:main": "src/index.js",
```

Comment:
I'm not following you.  The source appears to me to be ES2015, and tools that acknowledge `jsnext:main` can consume it as part of their transpilation pipelines.  So as far as my experience with rollup goes, I need to do nothing to the `src` and this will work fine.  Can you point to my misunderstanding?